packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -53,6 +53,11 @@ class FileCollection extends DocumentCollection {
     this.specialDirectories = {}
   }
 
+  /**
+   * get - Fetches the file's data
```

Comment:
```suggestion
   * Fetches the file's data
```

? 

I think the `get` is superfluous as it is picked up by jsdoc.

Reply:
Ah yes, it's generated by my IDE and assumed it would be deduplicated, but it should indeed not be here. Pushing again.